src/image/pixels.js

```diff
@@ -458,6 +463,44 @@ p5.prototype._copyHelper = (
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * createCanvas(100, 100, WEBGL);
+ * let myShader = createShader(
+ *   `attribute vec3 aPosition;
+ *   attribute vec2 aTexCoord;
+ *
+ *   varying vec2 vTexCoord;
+ *
+ *   void main() {
+ *     vTexCoord = aTexCoord;
+ *     vec4 positionVec4 = vec4(aPosition, 1.0);
+ *     positionVec4.xy = positionVec4.xy * 2.0 - 1.0;
+ *     gl_Position = positionVec4;
+ *   }`,
+ *   `precision mediump float;
+ *   varying mediump vec2 vTexCoord;
+ *
+ *   uniform sampler2D tex0;
+ *
+ *   float luma(vec3 color) {
+ *     return dot(color, vec3(0.299, 0.587, 0.114));
+ *   }
+ *
+ *   void main() {
+ *     vec2 uv = vTexCoord;
+ *     uv.y = 1.0 - uv.y;
+ *     vec4 sampledColor = texture2D(tex0, uv);
+ *     float gray = luma(sampledColor.rgb);
+ *     gl_FragColor = vec4(gray, gray, gray, 1);
+ *   }`
+ * );
+ * background('RED');
+ * filter(myShader);
```

Comment:
I think instead of just turning a red canvas grey it would be better to show off the feature but filtering an image that has been previously drawn